metatensor-core/src/io/labels.rs

```diff
@@ -1,17 +1,43 @@
+use std::io::BufReader;
+
 use byteorder::{LittleEndian, ReadBytesExt, BigEndian, WriteBytesExt, NativeEndian};
 
 use super::npy_header::{Header, DataType};
-use super::check_for_extra_bytes;
+use super::{check_for_extra_bytes, PathOrBuffer};
 use crate::{Error, Labels, LabelsBuilder, LabelValue};
 
+pub fn looks_like_labels_data(mut data: PathOrBuffer) -> bool {
```

Comment:
Maybe small doc what "looks like" means, not sure when `Header::from_reader(buffer)` is true

Reply:
done